modules/@apostrophecms/express/index.js

```diff
@@ -225,6 +225,12 @@ module.exports = {
         url: '/api/v1',
         middleware: cors()
       },
+      createRawRedirect(req, res, next) {
+        // We apply the super pattern variously to res.redirect,
+        // make sure the original version is always available
+        res.rawRedirect = res.redirect;
```

Comment:
Yes. If we tried to use `res.redirect` to `someDoc._url` on a site with no baseUrl, it would fail due to the prefixing. So we have always offered `rawRedirect` for such cases. At this very moment we are not using it in the core because I decided to handle locale switch URL changes in browser land javascript, but it wouldn't make sense not to offer this.

Reply:
It sounds like this is a loose thread for now, then. We should have a ticket to complete the loop.